src/axom/slic/tests/slic_macros.cpp

```diff
@@ -457,7 +456,7 @@ TEST(slic_macros, test_macros_file_output)
   std::string no_fmt_expected;
   no_fmt_expected += "*****\n[INFO]\n\n Test \n\n ";
   no_fmt_expected += __FILE__;
-  no_fmt_expected += "\n441\n****\n";
+  no_fmt_expected += "\n440\n****\n";
```

Comment:
What is `440` here? Is it a line number? 
It might make sense to add a comment about this for future maintenance of these tests.

Reply:
> What is 440 here? Is it a line number?

Yes. In a follow-up PR related to #1410, I'll be updating these line numbers to be like `slic_macros_parallel` to be `__LINE__ - offset` to make that clear.